doc/source/dev/contributor/pep8.rst

```diff
@@ -25,7 +25,7 @@ compliance before pushing your code:
 
    ::
 
-      flake8 scipy/optimize/linprog.py
+      flake8 scipy/optimize/_linprog.py
 
    The absence of output indicates that there are no PEP8 issues with
```

Comment:
I think this part can be delete. I remember thinking this was the case, but I think either the behavior has changed or it was never true.
```
(scipydev) Matts-MacBook-Pro:scipy matthaberland$ flake8 scipy/optimize/_linprog2.py
scipy/optimize/_linprog2.py:0:1: E902 FileNotFoundError: [Errno 2] No such file or directory: 'scipy/optimize/_linprog2.py'
```

Reply:
I think this part should be deleted regardless of what's above. It's just not true (anymore).

Reply:
Ohh I see, yeah this is not true anymore (I misunderstood before). Removing this section.

Reply:
Done.